sdk/core/azure_core_test/src/lib.rs

```diff
@@ -38,6 +38,8 @@ pub struct TestContext {
     recording: Option<Recording>,
 }
 
+unsafe impl Send for TestContext {}
```

Comment:
Would it be better to make `Recording` `Send`able? Either truly, or unsafely implementing `Send` like this for that? Because if so, then `TestContext` would be `Send` automatically. Looking at the `Recording` struct, I'm not sure exactly what makes it not `Send` (would be nice if there was a way to have the compiler tell us).